src/@types/d3/index.d.ts

```diff
@@ -0,0 +1,20 @@
+import { DSVRowAny, DSVRowString } from "d3";
```

Comment:
I'm guessing we copied this in from somewhere and if so, we should attribute the source at the top. If I'm wrong feel free to resolve this comment!

Reply:
no believe it or not I had to figure this shit out on my own 😒